### PR TITLE
New section in home on meower.org

### PR DESCRIPTION
**Pull request by roblox888i235**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,6 +58,12 @@ <h1>Open-Source</h1>
         all.</p>
     </section>
 
+    <section>
+      <h1>User Friendly</h1>
+      <p>We have a safe and welcoming community that will guide youu through Meower! <a href="https://app.meower.org"
```
Comment (JoshAtticus) [inline index.html:63]: youu

Comment (roblox888i235) [reply index.html:63]: what

Comment (JoshAtticus) [reply index.html:63]: pls read

Comment (JoshAtticus) [reply index.html:63]: not resolved, it says "will guide **youu**"

**Pull request closed without merging**